### PR TITLE
[bluetooth.airthings] Add support for Airthings Wave Radon

### PR DESCRIPTION
**Pull request by seime**
Add support for model Wave Radon https://www.airthings.com/en/wave-radon

### CODE REVIEW

**Review by jlaur — APPROVED** [review]
Thanks for the contribution! @paulianttila - would you like to have a look?

File: bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AirthingsWaveRadonHandler.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+                updateState(CHANNEL_ID_RADON_ST_AVG,
+                        new QuantityType<>(radonShortTermAvg, Units.BECQUEREL_PER_CUBIC_METRE));
+            }
+            Number radonLongTermAvg = data.get(AirthingsDataParser.RADON_LONG_TERM_AVG);
+            if (radonLongTermAvg != null) {
+                updateState(CHANNEL_ID_RADON_LT_AVG,
+                        new QuantityType<>(radonLongTermAvg, Units.BECQUEREL_PER_CUBIC_METRE));
+            }
+        } catch (AirthingsParserException e) {
+            logger.error("Failed to parse data received from Airthings sensor: {}", e.getMessage());
```
Comment (lsiepel) [inline bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AirthingsWaveRadonHandler.java:71]:
```suggestion
            logger.warn("Failed to parse data received from Airthings sensor: {}", e.getMessage());
```

File: bundles/org.openhab.binding.bluetooth.airthings/src/main/resources/OH-INF/thing/airthings.xml [hunk 2]
```diff
@@ -101,6 +101,39 @@
...
+			<bridge-type-ref id="bluegiga"/>
+			<bridge-type-ref id="bluez"/>
+		</supported-bridge-type-refs>
+
+		<label>Airthings Wave Radon / Wave 2</label>
+		<description>Indoor air quality monitor with radon detection</description>
+
+		<channels>
+			<channel id="rssi" typeId="rssi"/>
+
```
Comment (lsiepel) [inline bundles/org.openhab.binding.bluetooth.airthings/src/main/resources/OH-INF/thing/airthings.xml:116]:
```suggestion
```

**Review by lsiepel — CHANGES_REQUESTED** [review]
Otherwise LGTM.

File: bundles/org.openhab.binding.bluetooth.airthings/src/test/java/org/openhab/binding/bluetooth/airthings/AirthingsParserTest.java [hunk 3]
```diff
@@ -61,6 +63,19 @@ public void testParsingPlus() throws AirthingsParserException {
         assertEquals(122, result.get(AirthingsDataParser.RADON_SHORT_TERM_AVG));
     }
 
+    @Test
+    public void testParsingWaveRadon() throws AirthingsParserException {
+        // Testdata from
+        // https://github.com/Airthings/airthings-ble/blob/9d255808fa3add6d504649e40c8548ffcd356909/tests/test_wave_plus.py#L47
+        byte[] data = HexFormat.of().parseHex("013860f009001100a709ffffffffffff0000ffff");
```
Comment (paulianttila) [inline bundles/org.openhab.binding.bluetooth.airthings/src/test/java/org/openhab/binding/bluetooth/airthings/AirthingsParserTest.java:70]: Just to be consistent, would it make sense to use just an int array like it's used in other test?

**Review by paulianttila — APPROVED** [review]
Just one minor thing, otherwise LGTM

File: bundles/org.openhab.binding.bluetooth.airthings/src/test/java/org/openhab/binding/bluetooth/airthings/AirthingsParserTest.java (hunk 3 above)
Comment (seime) [reply bundles/org.openhab.binding.bluetooth.airthings/src/test/java/org/openhab/binding/bluetooth/airthings/AirthingsParserTest.java:70]: IMHO thus just shows the next dev it is easy to add testdata using data from the official AirThings repo that contains several other tests.

**Review by lsiepel — APPROVED** [review]

**Pull request merged**